packages/offchain-resolver/src/persistance/schema.prisma

```diff
@@ -13,6 +13,14 @@ model ProfileContainer {
   profile Json
   ensName String @unique
   address String @unique
+  aliases Alias[]
 
 }
 
+model Alias {
+    id Int @id @default(autoincrement())
```

Comment:
Do we need an additional ID when the namehash is part of the profile container ?

Reply:
I checked it it seems that the `String` wasn't the issue.